### ISSUE TITLE
Alexa Skills Stopped Working

### ISSUE THREAD

**Issue by joann235** [hardillb/node-red-contrib-alexa-home-skill#20]
I have been using the node-red alexa home skills bridge nodes on a Raspberry Pi 3 running Jessie.  About a week ago the skill stopped working.

My User Name is william235
I am running Node Red 0.16.2
The Alex node version is 0.1.15
Node JS is 6.11.2

When I go to discover devices it does not find any.  I have deleted and re-added devices on the web page and Alexa does not find them.    On the node red flows web page it shows that all the alexa nodes are connected.

I am at a complete loss and I would appreciate any help.

**Comment by hardillb** [comment]
Have you restarted Node-RED on the Pi before trying to select the devices?

**Comment by joann235** [comment]
Yes I have. Still finds no devices. All works fine through the node red ui.

Bill

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by hardillb** [comment]
OK, then I probably need to see the Node-RED log and the browser console log from when you press the refresh button in the node config dialog so I can see what errors it returns.

**Comment by joann235** [comment]
Ben:

As always, thanks for your ongoing assistance.  I do not know where to find
the logs you mentioned.  Can you please tell me what they would be called?

Thanks.

Bill

**Comment by joann235** [comment]
Ben:

In order to provide as much info as possible, here are some screen shots in
PDF format

Bill

**Comment by hardillb** [comment]
There is no PDF, github strips attachments when using email to update issues.

The browser console will depend which browser you are using but to get the console in Chrome you hit f12

As for the Node-RED logs it will depend on how you are running it. Assuming you are using the default service when it's installed via the node-red update script then you can tail the log with `node-red-log`

**Comment by joann235** [comment]
Ben:

I have made some progress but still not success.  Alexa now discovers 5 of
the 6 devices in my system.  When I say "Alexa Kitchen Off"  it says
"Kitchen is not responding".  Still can't find the reason why not.  Does
this give you any ideas as to what the problem might be?

I have no idea why it was suddenly able to discover the devices,

Bill

**Comment by hardillb** [comment]
Not immediately, if you can send me the flow that has the "Kitchen" object in then I can try and have a look. If you high light all the nodes in the flow that are linked to the Kitchen node then pick export -> clipboard from the menu in the top right hand corner you should be able to copy and past the json for just those nodes and paste them in here so I can check a few things.

**Comment by joann235** [comment]
Ben:

Happy (and a little embarrassed) to say that I found the problem.  In order
to save steps walking from my office to the echo dot I was using my IPhone
to login to my account.  The IPhone capitalized the first letter of my user
name and thus was not finding any devices.  When I logged in in all lower
case for my user name it discovered all six devices and began working.  I
honestly did not know that case mattered in a user name.

I hate to think that I wasted your time and I appreciate your willingness
to help.  Several weeks ago I praised your product on Pete Scargills site
as being easy to use and very good quality.  I still feel that way.

Regards,
Bill Miller
